tox.ini

```diff
@@ -25,6 +25,7 @@ deps =
     1.8: Django>=1.8,<1.9
     1.9: Django>=1.9,<1.10
     1.10: Django>=1.10,<1.11
+    1.11: Django==1.11a1
```

Comment:
Might want to use `>=1.11a1,<2.0` here so we don't have to adjust every alpha/beta and rc releases.